e2e/wdio/platform/tests/wizard-generator.e2e-spec.ts

```diff
@@ -0,0 +1,545 @@
+        it('should check basic way', () => {
+            click(customizableDialogExample + button);
+            checkBasicWay(dialog);
+        });
+
+        it('should check required fields validation', () => {
+            click(customizableDialogExample + button);
+            checkValidationRequiredFields(dialog);
+        });
+
+        it('should check navigation by steps', () => {
+            click(customizableDialogExample + button);
+            checkNavigationBySteps(dialog);
+        });
+
+        it('should check navigation by buttons', () => {
+            checkNavigationByButtonsInDialog(customizableDialogExample);
+        });
+    });
+
+    describe('condition dialog example', () => {
+        it('should check basic way', () => {
+            click(branchingExample + button);
+            click(dialog + select);
+            const productType = getText(listItemText);
+            click(listItem);
+            click(dialog + nextStepBtn2);
+
+            expect(getElementClass(dialog + step, 0)).toContain('completed', 'first step is not completed');
+            expect(getElementClass(dialog + step, 1)).toContain('current', 'you not moved to second step');
+            setValue(dialog + input, name);
+            setValue(dialog + input, firstAdress, 1);
+            setValue(dialog + input, secondAdress, 2);
+            click(dialog + nextStepBtn2);
+            expect(getElementClass(dialog + step, 1)).toContain('completed', 'second step is not completed');
+            expect(getElementClass(dialog + step, 2)).toContain('current', 'you not moved to third step');
+            click(dialog + select);
+            const paymentMethod = getText(listItemText);
+            click(listItem);
+            click(dialog + nextStepBtn2);
+            setValue(dialog + input, cardDetails);
+            click(dialog + nextStepBtn2);
+            click(dialog + nextStepBtn2);
+
+            expect(getText(dialog + formLabel, 0)).toBe(productType, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 1)).toBe(name, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 2)).toBe(firstAdress, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 3)).toBe(secondAdress, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 4)).toBe(paymentMethod, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 5)).toBe(cardDetails, 'value is not equal entered value');
+            expect(getText(dialog + formLabel, 6)).toBe('-', 'no dash');
+            click(dialog + editButton, 3);
+            setValue(dialog + input, cardDetails2);
+            click(dialog + nextStepBtn2);
+            expect(getText(dialog + formLabel, 5)).toBe(cardDetails2, 'value is not equal changed value');
+        });
+
+        it('should check required fields validation', () => {
```

Comment:
could also break this one down into a couple test cases so we will know better if all validation or 1 validation broken

it should check step 1 validation
...
it should check step 2 validation
completeStepOne()
click(nextBtn)
..add invalid text
assert..
...